### PR TITLE
Fix submenu link highlighting

### PR DESCRIPTION
**Pull request by develohpanda**
### Summary
Updated a few things:
* Used [`{% break %}`](https://shopify.github.io/liquid/tags/iteration/#break) to ensure the `show` class is only added once
* renamed `currentPage` to `currentEntry` to avoid confusion
* updated array [size](https://shopify.github.io/liquid/filters/size/) check
* updated the styling to be consistent with the rest of the nav, `ps-3` and (effectively) `pb-1`

### Reason

Follow up to #35 

### Testing

Use the sub nav menu under Inso CLI -> CLI Command Reference

image

### CODE REVIEW

File: docs/_includes/sidebar-nav.html
Comment (develohpanda) [inline docs/_includes/sidebar-nav.html:19]: What is the `id` attribute in this div used for? `subentry.collapse-id` doesn't exist in this context so it just outputs `<div id="-collapse" ... >`

Comment (falondarville) [reply docs/_includes/sidebar-nav.html:19]: This can be removed. Were you planning to merge this right after my PR is merged?

**Review by falondarville — APPROVED** [review]
LGTM. Left one comment.

File: docs/_includes/sidebar-nav.html
Comment (develohpanda) [reply docs/_includes/sidebar-nav.html:19]: Ah was going to let you merge it into your PR. No matter, I'll rebase onto main and merge it there.

**Pull request merged**
